src/main/java/com/Elessar/app/server/DirectMsgSender.java

```diff
@@ -7,8 +7,11 @@
 import com.Elessar.proto.P2Pmsg.P2PMsgRequest;
 import com.Elessar.proto.P2Pmsg.P2PMsgResponse;
 import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
```

Comment:
not sure why we need this and 
import org.apache.tools.ant.taskdefs.condition.Http;